docs/content/toc.textile

```diff
@@ -64,6 +64,7 @@ h2. Operations
 * "Different Hadoop Versions":../operations/other-hadoop.html
 * "Multitenancy Considerations":../operations/multitenancy.html
 * "Performance FAQ":../operations/performance-faq.html
+* "Pull Dependencies":../operations/pull-deps.html
```

Comment:
I don't believe this deserves a TOC entry and the file should be linked via the "Including Extensions" page